tree/dataframe/src/RDataFrame.cxx

```diff
@@ -1324,9 +1324,9 @@ RDataFrame::RDataFrame(std::string_view treeName, std::string_view filenameglob,
 {
    const std::string treeNameInt(treeName);
    const std::string filenameglobInt(filenameglob);
-   auto chain = std::make_shared<TChain>(treeNameInt.c_str());
+   auto chain = std::make_shared<TChain>(treeNameInt.c_str(), "", TChain::kWithoutGlobalRegistration);
```

Comment:
This might be the trigger of the current failures.

```
#6  0x000055b2f02ff9e0 in ?? ()
#7  0x00007f59ede26d32 in TTree::GetReadCache(TFile*) const () from /data/sftnight/workspace/root-pullrequests-build/build/lib/libTree.so.6.27
#8  0x00007f59edde960f in TChain::~TChain() () from /data/sftnight/workspace/root-pullrequests-build/build/lib/libTree.so.6.27
#9  0x00007f5a011e3bc7 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() () from /data/sftnight/workspace/root-pullrequests-build/build/lib/libRIO.so
#10 0x00007f59ec7b1684 in ROOT::Detail::RDF::RLoopManager::~RLoopManager() () from /data/sftnight/workspace/root-pullrequests-build/build/lib/libROOTDataFrame.so.6.27.01
```

I.e. When not registered the `TChain` is not informed of deleted objects and thus may not be informed the TFile was deleted ....

Reply:
thank you for taking a look, i'll ask valgrind :)

Reply:
I removed the `TChain::kWithoutGlobalRegistration` from these TChains, I'll readd it after #10742 is fixed